bin/securedrop

```diff
@@ -15,4 +16,23 @@ if __name__ == "__main__":
         if not os.path.exists(securedrop_dir):
             raise RuntimeError("Failed to create securedrop directory: $HOME/.securedrop")
     securedrop_file = os.path.join(securedrop_dir, "client_db.json")
-    client.main(filename=securedrop_file)
+    securedrop_port = None
+    securedrop_hostname = None
+    verbose_flag = False
+    try:
+        opts, args = getopt.getopt(sys.argv[1:], "pafv", ["port", "address", "filename", "verbose"])
+    except getopt.GetoptError as err:
+        print(err)  # will print something like "option -a not recognized"
+        sys.exit(2)
+    for o, a in opts:
+        if o in ("-p", "--port"):
+            securedrop_port = a
+        elif o in ("-f", "--filename"):
+            securedrop_file = os.path.join(securedrop_dir, a)
+        elif o in ("-a", "--address"):
+            securedrop_hostname = a
+        elif o in ("-v", "--verbose"):
+            verbose_flag = True
+        else:
+            assert False, "unhandled option"
```

Comment:
I'm not a big fan of asserts used like this. Assert is typically used for sanity checks, whereas this usage is more appropriate for some kind of exception. Please find an appropriate exception type to throw (when in doubt throw RuntimeError).

Reply:
OK, I couldn't find anything that seemed more appropriate than runtime error.